Formula/diffuse.rb

```diff
@@ -0,0 +1,29 @@
+class Diffuse < Formula
+  desc "Graphical tool for merging and comparing text files"
+  homepage "https://github.com/MightyCreak/diffuse"
+  url "https://github.com/MightyCreak/diffuse/archive/refs/tags/v0.7.3.tar.gz"
+  sha256 "cceff03beabc83fc76c49ecdad6743b7bd709063167af851fe2ebae7a10b1394"
+  license "GPL-2.0-or-later"
+
+  depends_on "gettext" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+
+  depends_on "gtk+3"
+  depends_on "pygobject3"
+  depends_on "python@3.9"
+
+  def install
+    ENV["DESTDIR"] = "/"
```

Comment:
I'm suspicious about needing this.

Reply:
code ref, https://github.com/MightyCreak/diffuse/blob/ea231a344482f6c16c5d10b21c5c685e2e576cfe/build-aux/meson/postinstall.py#L10-L19